.github/workflows/test.yml

```diff
@@ -107,68 +107,72 @@ jobs:
           test -f debian/artifacts/test_1_amd64.buildinfo
           test -f debian/artifacts/test_1_amd64.changes
 
-  local-dockerfile:
+  extra-docker-args:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: cat test/Makefile_local-dockerfile >> test/Makefile
+      - run: cat test/Makefile_extra-docker-args >> test/Makefile
       - uses: ./
-        env:
+        env: 
           DEB_BUILD_OPTIONS: noautodbgsym
         with:
           buildpackage-opts: --build=binary --no-sign
-          docker-image: test/Dockerfile
           source-dir: test
+          extra-docker-args: --add-host extra-args-test:127.0.0.1
       - run: |
+          test "$(ls -1 debian/artifacts/*.deb | wc -l)" = 1
```

Comment:
Can also drop this line here. If there is multiple packages by mistake, the `single-package` test will catch that. Make this one purely about the docker-args.
```suggestion
```